Feature/44067

Aekal opened the pull request:
dev-44067
#384

anqaka commented on components/02-elements/swatch/swatch.config.js:
```diff
@@ -27,51 +24,33 @@ module.exports = {
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-white.jpg);"'
           }
         ]
```
I've updated the config with aria attributes, please implement it where needed (views, modules)

anqaka commented on components/03-modules/brief-info/brief-info.hbs:
```diff
@@ -2,19 +2,21 @@
+    <div class="brief-info__items-container">
+        {{#each items as |item|}}
+            <div class="brief-info__item">
+                <span class="brief-info__icon-wrapper {{ item.iconWrapperClass }}">
```
missing `aria-label` (there is no text, only icon inside

Aekal replied:
Added

anqaka commented on components/03-modules/brief-info/brief-info.hbs:
```diff
@@ -2,19 +2,21 @@
-        </div>
+    <div class="brief-info__items-container">
+        {{#each items as |item|}}
+            <div class="brief-info__item">
```
maybe we can put it in list? b/c in fact we list here some features...

Aekal replied:
Done

anqaka commented on components/04-views/product-view/_product-view.scss:
```diff
@@ -260,13 +262,19 @@ $product-view__wrapper-margin\@medium                   : 0 0 40px 0 !default;
+            margin-bottom: $product-view__swatch-margin-bottom;
+            &__option {
+                min-width: $product-view__swatch-option-spacing;
+                min-height: $product-view__swatch-option-spacing;
```
interactives e,ements should be at least 48 x 48px of clickable area

Aekal replied:
Done

Aekal commented on components/02-elements/swatch/_swatch.scss:
```diff
@@ -4,6 +4,7 @@ $swatch__title-margin-bottom         : $spacer !default;
 $swatch__title-font-size             : $font-size-base !default;
 
 $swatch__option-size                 : 24px !default;
+$swatch__option-size--image          : 32px !default;
```
I made them bigger, each swatch has now 48x48px of clickable area.

Aekal commented:
Thanks for the readme, I've added accordingly aria attributes with script to simulate how do they work in magento.
I also added more variants of swatch: default, image, color (+ with labels) and I made some adjustments to `catalog-grid-item` (fixed spacing and added gray border - IMO looks better now).
Changelog updated.

anqaka commented:
Great job! I don't have a pictures on local and I hear everything well 😍 👍
